classes/class-wc-connect-shipping-label.php

```diff
@@ -470,6 +464,18 @@ public function meta_box( $post ) {
 			$store_options['countriesData'] = $this->get_states_map();
 			$payload['storeOptions'] = $store_options;
 
+			return $payload;
+		}
+
+		public function meta_box( $post ) {
+			$order = wc_get_order( $post );
+
+			$order_id = WC_Connect_Compatibility::instance()->get_order_id( $order );
+
+			$payload = $this->get_label_payload( $order );
+			//pass order ID as the only constant in the context, everything else can be fetched dynamically
+			$payload[ 'orderId' ] = $order_id;
```

Comment:
I'm not sure I understand. It looks like `orderId` is always in the returned payload from `get_label_payload()`, so what's the point in overwriting it here?